### PR TITLE
More stats data

### PR DESCRIPTION
**Pull request by dswcodes0**
i added more stats data into the home page
Screenshot 2023-07-24 at 11 55 28 AM

### CODE REVIEW

**Comment by vqdo** [comment]
What about wins/losses or the K/D ratio? Does that apply for this game?

(Also.... what game is this? :D Could you include that somewhere in the UI?)

File: statchatui/src/Stats/Stats.jsx [hunk 1]
```diff
@@ -1,13 +1,21 @@
 import React from "react";
 
 const Stats = ({ statData }) => {
+  console.log(statData);
   const kills = statData?.total?.kills?.value;
+  const level = statData?.global?.level;
+  const name = statData?.global?.name;
+  const rank = statData?.global?.rank?.rankName;
```
Comment (vqdo) [inline statchatui/src/Stats/Stats.jsx:8]: Let's try to handle the null state. ex:

```
if (statData == null) {
  return <div>UI shown when we don't have any data yet</div>
}

// Now we can assume that statData exists
const kills = statData.total?.kills?.value;
....

```

**Review by vqdo — CHANGES_REQUESTED** [review]

**Comment by dswcodes0** [comment]
> What about wins/losses or the K/D ratio? Does that apply for this game?
> 
> (Also.... what game is this? :D Could you include that somewhere in the UI?)

this game is apex legends

**Comment by dswcodes0** [comment]
> > What about wins/losses or the K/D ratio? Does that apply for this game?
> > (Also.... what game is this? :D Could you include that somewhere in the UI?)
> 
> this game is apex legends

this game does not have data on deaths so i cant do k/d. i cant do wins either

File: statchatui/src/Stats/Stats.jsx [hunk 2]
```diff
@@ -1,16 +1,28 @@
...
+          <h1>Apex Legends</h1>
+          <h2>Stats:</h2>
+          <h3>Name: {name}</h3>
+          <h3>Level: {level}</h3>
+          <h3>Rank: {rank}</h3>
+          <h3>Kills: {kills}</h3>
         </div>
       ) : (
-        <div>Loading...</div>
+        <div>Stats not yet fetched.</div>
```
Comment (vqdo) [inline statchatui/src/Stats/Stats.jsx:24]: Looks good. To make this a little cleaner we could return early so we only need to check `statData` once!

**Review by vqdo — APPROVED** [review]

**Pull request merged**